content/version/1/4/code-of-conduct.md

```diff
@@ -10,9 +10,9 @@ aliases = ["/version/1/4"]
 In the interest of fostering an open and welcoming environment, we as
 contributors and maintainers pledge to making participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-education, socio-economic status, nationality, personal appearance, race,
-religion, or sexual identity and orientation.
+size, disability, ethnicity, gender (sex), gender identity and expression, level
```

Comment:
If you changed it to "ethnicity, sex, gender, …" I would be happy and we could merge.

Reply:
Oh, I just noticed that you listed gender alone additionally in your example above ("ethnicity, sex, gender, ..."). Was that a typo, or do you think gender should also be on the list separately from both sex and gender identity? (To be honest I don't know what it would refer to in that case.)